### DISCUSSION TITLE
Just invited @PatheticMustan to the PMGH blog

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#891] [(D) Developer]
@PatheticMustan i just emailed you back about the PMGH Blog and invited you to the repository, is there any other moderator you want me to add?(ArcerionDev, ChickenOnYT etc.)

**Comment by PatheticMustan** [comment]
on the contrary, just invited you to one of the PMGH repos

**Reply by SakuraKitsunee** [reply]
i just got that is that where we are going to make the blog??

**Reply by SakuraKitsunee** [reply]
We could also add the one I made to the Prodigy Hacking organization, if you want.

**Reply by SakuraKitsunee** [reply]
@PatheticMustan

**Comment by PatheticMustan** [comment]
but... you don't have anything pushed yet...??

**Reply by SakuraKitsunee** [reply]
@PatheticMustan true, are we going to delete the other stuff, like the status page and stuff like that or no?

**Reply by ghost** [reply]
@SakuraKitsunee

**Reply by PatheticMustan** [reply]
sure

**Comment by ghost** [comment]
@PatheticMustan

**Reply by NX721** [reply]
Ezz Don't ping

**Reply by ghost** [reply]
yes
